wgsl/index.bs

```diff
@@ -1866,7 +1866,7 @@ and when the storage class decoration is required, optional, or forbidden.
 variable_statement
   : variable_decl
   | variable_decl EQUAL short_circuit_or_expression
-  | CONST variable_ident_decl EQUAL short_circuit_or_expression
+  | CONST (IDENT | variable_ident_decl) EQUAL short_circuit_or_expression
```

Comment:
The 'variable_ident_decl' production also starts with IDENT.

I think there are already other shortcuts like this in the grammar, so we can revisit that if/when we get a mechanically-processable grammar.